docs/en/table_design/table_types/primary_key_table.md

```diff
@@ -1,159 +1,212 @@
+### Primary key
 
-Example 2: Suppose that you need to analyze user behavior in real time from dimensions such as users' address and last active time. When you create a table, you can define the `user_id` column as the primary key and define the combination of the `address` and `last_active` columns as the sort key.
+The primary key of a table is used to uniquely Identify each row in that table. The one or more columns comprising the primary key are defined in the `PRIMARY KEY`, and have the UNIQUE constraint and NOT NULL constraint.
 
-```SQL
-create table users (
-    user_id bigint NOT NULL,
-    name string NOT NULL,
-    email string NULL,
-    address string NULL,
-    age tinyint NULL,
-    sex tinyint NULL,
-    last_active datetime,
-    property0 tinyint NOT NULL,
-    property1 tinyint NOT NULL,
-    property2 tinyint NOT NULL,
-    property3 tinyint NOT NULL,
-    ....
-) PRIMARY KEY (user_id)
-DISTRIBUTED BY HASH(user_id)
-ORDER BY(`address`,`last_active`)
-PROPERTIES("replication_num" = "3",
-"enable_persistent_index" = "true");
-```
+Take note of the following considerations about the primary key:
+
+- In the CREATE TABLE statement, the primary key columns must be defined before other columns.
+- The primary key columns must include partitioning and bucketing columns.
+- The primary key columns support the following data types: numeric (including integers and BOOLEAN), string, and date (DATE and DATETIME).
+- The maximum length of a encoded primary key value is 128 bytes.
+- The primary key cannot be modified after table creation.
+- For data consistency purposes, the primary key values cannot be updated.
+
+### Primary key index
+
+The primary key index is used to store the mapping between the primary key values and the locations of the data rows identified by the primary key values. Typically, the primary key indexes of relevant tablets are loaded into memory only during data loading (which involves a batch of data changes). You can consider persisting the primary key indexes after comprehensively evaluating the performance requirement for queries and updates, as well as the memory and disk.
+
+<Tabs groupId="primary key Index">
+
+  <TabItem value="example1" label="Persistent primary keyIndex" default>
+
+When `enable_persistent_index` is set to `true` (default), the primary key indexes can be persisted to the disk. During loading, a small portion of the primary key indexes is loaded in memory, while the majority is stored on disk to avoid taking up too much memory. In general, query and update performance of the table with the persistent primary key indexes is nearly equivalent to that of the table with the fully in-memory primary key indexes.
+
+If the disk is a SSD, it is recommended to set it to true. If the disk is HDD and the load frequency is not high, you can also set it to true.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8e8097b5-9a41-4750-8067-e7aad0ec3bb7)
1. wrap the two "true" in code format.
2. HDD -> an HDD